github/enterprise_actions_permissions.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ActionsPermissionsEnterprise represents a policy for allowed actions in an enterprise.
+//
+// GitHub API docs: https://docs.github.com/en/rest/actions/permissions
+type ActionsPermissionsEnterprise struct {
+	EnabledOrganizations *string `json:"enabled_organizations,omitempty"`
+	AllowedActions       *string `json:"allowed_actions,omitempty"`
+	SelectedActionsURL   *string `json:"selected_actions_url,omitempty"`
+}
+
+func (a ActionsPermissionsEnterprise) String() string {
+	return Stringify(a)
+}
+
+// ActionsEnabledOnEnterpriseOrgs represents all the repositories in an enterprise for which Actions is enabled.
+type ActionsEnabledOnEnterpriseOrgs struct {
+	TotalCount    int             `json:"total_count"`
+	Organizations []*Organization `json:"organizations"`
+}
```

Comment:
```suggestion
// ActionsEnabledOnEnterpriseOrgs represents all the repositories in an enterprise for which Actions is enabled.
type ActionsEnabledOnEnterpriseOrgs struct {
	TotalCount    int             `json:"total_count"`
	Organizations []*Organization `json:"organizations"`
}

// ActionsPermissionsEnterprise represents a policy for allowed actions in an enterprise.
//
// GitHub API docs: https://docs.github.com/en/rest/actions/permissions
type ActionsPermissionsEnterprise struct {
	EnabledOrganizations *string `json:"enabled_organizations,omitempty"`
	AllowedActions       *string `json:"allowed_actions,omitempty"`
	SelectedActionsURL   *string `json:"selected_actions_url,omitempty"`
}

func (a ActionsPermissionsEnterprise) String() string {
	return Stringify(a)
}
```

This is kind of hacky, but it clears the duplicate code issue.